apiserver/params/params.go

```diff
@@ -1012,6 +1012,12 @@ type FindToolsResult struct {
 	Error *Error     `json:"error,omitempty"`
 }
 
+// ToolVersionsResult holds a list of tool versions and any error.
+type ToolVersionsResult struct {
+	ToolVersions tools.List `json:"tool-versions"`
+	Error        *Error     `json:"error,omitempty"`
+}
+
```

Comment:
There's already 2 versions of this struct - ToolsResult and FindToolsResult
(there should only be one, not sure why there's 2)

Reply:
Yeah, they were initially different, but now they are similar. Changed to use FindToolsResult